src/libraries/Common/src/Interop/Unix/Interop.Libraries.cs

```diff
@@ -7,12 +7,12 @@ internal static partial class Interop
     internal static partial class Libraries
     {
         // Shims
-        internal const string SystemNative = "System.Native";
-        internal const string GlobalizationNative = "System.Globalization.Native";
-        internal const string NetSecurityNative = "System.Net.Security.Native";
-        internal const string CryptoNative = "System.Security.Cryptography.Native.OpenSsl";
-        internal const string CompressionNative = "System.IO.Compression.Native";
-        internal const string IOPortsNative = "System.IO.Ports.Native";
+        internal const string SystemNative = "libSystem.Native";
+        internal const string GlobalizationNative = "libSystem.Globalization.Native";
+        internal const string NetSecurityNative = "libSystem.Net.Security.Native";
+        internal const string CryptoNative = "libSystem.Security.Cryptography.Native.OpenSsl";
```

Comment:
This change, overall, makes me sad.  We intentionally stripped off the lib prefix as part of "these aren't general purpose libraries, don't look here".  But if we need to, we need to, I guess.

Reply:
The alternative may be to statically link these libraries into the runtime for Android where it is causing problems. We are going to have this option for single file flavor of CoreCLR anyway, so doing it for Android as well should not be a big deal.

Reply:
CC @akoeplinger and @marek-safar to be aware of the suggestion.

Reply:
>statically link these libraries into the runtime
Xamarin.Android does not support this mode. I think this would be nice but we won't be in a position to deliver this for .net5. This looks more like .net6 enhancement because we would need to do native linking during publish phase (this depends on Android NDK, etc).

/cc @jonpryor

Reply:
For CoreCLR, the plan is to statically link the libraries as part of the dotnet/runtime build (see https://github.com/dotnet/runtime/issues/32823). We are not planning to do native linking during publish phase.

Reply:
That won't work for us. The thinking right now is to copy the dynamic libs which are used (during publish) and extend that to full static linking for Android as we plan to do for iOS (and wasm) during publish

Reply:
Just curious - why it does not work for you?

Reply:
Primary because of size. If an app uses no crypto we should not link `libSystem.Net.Security.Native` and you can apply that to all native libs where `System.IO.Compression.Native` is probably the biggest one.

Reply:
The moment someone brings in any managed HTTP implementation, client or server, it's going to end up requiring both of those.  Presumably then the size concern would only be if they were using a platform-specific implementation (e.g. AndroidHttpHandler) where the platform provided all of that support?